vector/v.to.rast/vect2rast.c

```diff
@@ -149,6 +149,19 @@ int vect_to_rast(const char *vector_map, const char *raster_map,
         dense = 0;
     }
 
+    if (ftype & GV_POINT && !Vect_get_num_primitives(&Map, GV_POINT)) {
+        G_warning(_("No points selected from vector map <%s>"), vector_map);
+    }
+    if (ftype & GV_LINE && !Vect_get_num_primitives(&Map, GV_LINE)) {
+        G_warning(_("No lines selected from vector map <%s>"), vector_map);
+    }
+    if (ftype & GV_BOUNDARY && !Vect_get_num_primitives(&Map, GV_BOUNDARY)) {
+        G_warning(_("No boundaries selected from vector map <%s>"), vector_map);
+    }
+    if (ftype & GV_CENTROID && !Vect_get_num_primitives(&Map, GV_CENTROID)) {
+        G_warning(_("No centroids selected from vector map <%s>"), vector_map);
```

Comment:
Since #2459 was merged, the message for areas is now "Number of areas selected from vector map <%s>: %d". So, merging this wouldn't be consistent anymore. Would a similar adaptation should be done? Is the reasoning that lead to the change in #2459 still applies to the other types?

Reply:
Yes, as the comment from August 2022 alluded to, this modification makes sense in terms of achieving consistency. #2459 achieved consistency in producing no warnings. This PR is not adding consistency on verbose message level.